queries/crowdstrike_queries/CrowdStrike_Large_Zip_Creation.yml

```diff
@@ -0,0 +1,65 @@
+AnalysisType: scheduled_query
+Description: Detects creation of large zip files, which can indicate attempts of exfiltration
+Enabled: false
+Query: "
```

Comment:
in the place of `Query: "` 

it would be more yaml-like to open with `Query: |`

Yaml will interpret whatever follows to be a string that contains newlines, provided that the following text is indented. Yaml will consider the next start-of-line-is-character line to be where the `|` newline-containing string ends.  examples here https://github.com/panther-labs/panther-analysis/blob/master/queries/aws_queries/anomalous_access_denied_query.yml#L4-L14